### PR TITLE
WIP: add stats command

### PR DESCRIPTION
**Pull request by vharitonsky**

### CODE REVIEW

File: src/lib/parser.ts [hunk 1]
```diff
@@ -24,8 +24,13 @@ export type Translations = {
     [key: string]: Messages;
 };
 
+export type Headers = {
+    "content-type"?: string;
+    "plural-forms"?: string;
```
Comment (AlexMost) [inline src/lib/parser.ts]: `plural-forms` is the 'must have' header

Comment (vharitonsky) [reply src/lib/parser.ts]: unfortunately gettext-parser does not parse them, gotta do some postprocessing

Comment (AlexMost) [reply src/lib/parser.ts]: so sad

**Comment by AlexMost** [comment]
👍

**Pull request merged**